components/raftstore/src/store/worker/pd.rs

```diff
@@ -191,15 +195,20 @@ impl Default for StoreStat {
 pub struct PeerStat {
     pub read_bytes: u64,
     pub read_keys: u64,
+    pub read_query_stats: QueryStats,
     // last_region_report_attributes records the state of the last region heartbeat
     pub last_region_report_read_bytes: u64,
     pub last_region_report_read_keys: u64,
+    pub last_region_region_read_query_stats: QueryStats,
     pub last_region_report_written_bytes: u64,
     pub last_region_report_written_keys: u64,
+    pub last_region_report_written_query_num: u64,
+    pub last_region_report_query_stats: u64,
     pub last_region_report_ts: UnixSecs,
     // last_store_report_attributes records the state of the last store heartbeat
     pub last_store_report_read_bytes: u64,
     pub last_store_report_read_keys: u64,
+    pub last_store_report_read_query_stats: QueryStats,
```

Comment:
I don't think maintaining separate `read/write` query_stats looks good as `query_stats` already contains `put` here.
Maybe only maintaining `last_query_stats` is enough.